superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -861,19 +925,98 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     }
   }, [availableDbs]);
 
-  const tabChange = (key: string) => {
-    setTabKey(key);
+  const onDbImport = (info: UploadChangeParam) => {
+    setImportingModal(true);
+    setFile([
+      {
+        ...info.file,
+        status: 'done',
+      },
+    ]);
+
+    if (!(info.file.originFileObj instanceof File)) return;
+    importResource(info.file.originFileObj, passwords, confirmedOverwrite);
+  };
+
+  const passwordNeededField = () => {
+    if (passwordsNeeded.length === 0) return null;
+
+    return passwordsNeeded.map(database => (
+      <>
+        <StyledAlertMargin>
+          <Alert
+            closable={false}
+            css={(theme: SupersetTheme) => antDAlertStyles(theme)}
+            type="info"
+            showIcon
+            message="Database passwords"
+            description={t(
+              `The passwords for the databases below are needed in order to import them. Please not that the "Secure Extra" and "Certificate" sections of the database configuration are not present in explore files and should be added manually after the import if they are needed.`,
```

Comment:
```suggestion
              `The passwords for the databases below are needed in order to import them. Please note that the "Secure Extra" and "Certificate" sections of the database configuration are also not present in exported files and should be added manually after the import if they are needed.`,
```

Reply:
Whoops, thank you for this catch! Corrected in [`this commit`](https://github.com/apache/superset/pull/19314/commits/cb6ef83987c197fee8f24f88de82730b569e4a22).